src/components/screens/lockedBalance/lockedBalance/form.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { withTranslation } from 'react-i18next';
+import to from 'await-to-js';
+import Box from '../../../toolbox/box';
+import BoxContent from '../../../toolbox/box/content';
+import BoxFooter from '../../../toolbox/box/footer';
+import BoxHeader from '../../../toolbox/box/header';
+import { PrimaryButton } from '../../../toolbox/buttons';
+import { toRawLsk, fromRawLsk } from '../../../../utils/lsk';
+import Piwik from '../../../../utils/piwik';
+import { getAvailableUnlockingTransactions } from '../../../../utils/account';
+import { create } from '../../../../utils/api/lsk/transactions';
+import transactionTypes from '../../../../constants/transactionTypes';
+import actionTypes from '../../../../constants/actions';
+import styles from './lockedBalance.css';
+
+const Form = ({
+  t,
+  children,
+  nextStep,
+  data,
+}) => {
+  const {
+    account,
+    customFee,
+    fee,
+    currentBlock,
+    availableBalance,
+  } = data;
+  const dispatch = useDispatch();
+  const network = useSelector(state => state.network);
+
+  const onClickUnlock = async () => {
+    Piwik.trackingEvent('Send_SubmitTransaction', 'button', 'Next step');
```

Comment:
We use `Send_SubmitTransaction` to keep the stats of the send form. This is a different tx.